api/controller/xtra_db_operator_api.proto

```diff
@@ -20,6 +20,12 @@ message InstallXtraDBOperatorRequest {
       msg_exists: true
     }
   ];
+  // Operator version to be installed.
+  string version = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
 }
 
 message InstallXtraDBOperatorResponse {}
```

Comment:
Either we return err or nil, so we don't really need status. The only thing that can fail is when we fail to reach github for manifests we are trying to install.